gettsim/tests/test_data/test_dfs_eltg.csv

```diff
@@ -0,0 +1,20 @@
+hid,tu_id,pid,m_wage,m_wage_l1,east,child,incometax,soli,svbeit,byear,bday,bmonth,age,elterngeld_mon_mut,elterngeld_mon_vat,elterngeld_mon,,year,,,,elternzeit_anspruch,geschw_bonus,num_mehrlinge,elterngeld,considered_wage,payed_percentage
+1,1,23,0,0,FALSE,FALSE,0,0,0,1972,1,1,46,0,0,0,,2018,,,,FALSE,FALSE,0,0,,
+1,1,23,0,0,FALSE,FALSE,0,0,0,1972,2,2,46,0,0,0,,2019,,,,TRUE,FALSE,0,300,0,0
+1,1,18,0,0,FALSE,TRUE,0,0,0,2018,12,10,0,2,2,0,,2019,,,,FALSE,FALSE,0,0,,
+2,2,42,1800,1800,FALSE,FALSE,218.96,12.04,378,1974,3,3,42,0,0,0,,2017,,,,FALSE,FALSE,0,0,,
+2,2,42,100,1800,FALSE,FALSE,0,0,0,1974,4,4,43,0,0,0,,2018,,,,TRUE,FALSE,0,730.97,1091,0.67
+2,2,43,0,0,FALSE,TRUE,0,0,0,2017,12,6,0,1,0,0,,2018,,,,FALSE,FALSE,0,0,,
+3,3,55,900,900,FALSE,FALSE,7.68,0,180,1976,5,5,41,0,0,0,,2018,,,,FALSE,FALSE,0,0,,
+3,3,55,0,900,FALSE,FALSE,0,0,0,1976,6,6,42,0,0,0,,2019,,,,TRUE,FALSE,0,575.554889,703.32,0.81834
+3,3,50,0,0,FALSE,TRUE,0,0,0,2018,12,6,0,1,0,0,,2019,,,,FALSE,FALSE,0,0,,
+3,3,56,3400,3600,FALSE,FALSE,708.27,38.96,680,1976,7,7,41,0,0,0,,2018,,,,FALSE,FALSE,0,0,,
+3,3,56,3600,3400,FALSE,FALSE,788.34,43.36,756,1976,8,8,42,0,0,0,,2019,,,,TRUE,FALSE,0,0,,
+4,4,59,0,0,FALSE,TRUE,0,0,0,2017,12,6,0,1,0,0,,2018,,,,FALSE,FALSE,0,0,,
+4,4,57,3600,3400,FALSE,FALSE,779.33,42.86,756,1976,9,9,41,0,0,0,,2018,,,,TRUE,FALSE,0,0,,
+5,5,58,0,3600,FALSE,FALSE,0,0,0,1978,10,10,40,0,0,0,,2019,,,,TRUE,FALSE,0,1314.1765,2021.81,0.65
+5,5,60,0,0,FALSE,TRUE,0,0,0,2018,12,4,0,1,0,0,,2019,,,,FALSE,FALSE,0,0,,
+6,19,19,900,900,FALSE,FALSE,7.68,0,180,1980,11,11,37,0,0,0,,2018,,,,FALSE,FALSE,0,0,,
+6,19,20,0,0,FALSE,TRUE,0,0,0,2017,12,2,1,0,0,0,,2019,,,,FALSE,FALSE,0,0,,
+6,19,21,0,0,FALSE,TRUE,0,0,0,2018,12,10,0,2,2,0,,2019,,,,FALSE,FALSE,0,0,,
+6,19,19,0,900,FALSE,FALSE,0,0,0,1982,13,2,36,0,0,2,,2019,,,,TRUE,TRUE,0,650.554889,703.32,0.81834
```

Comment:
can we have at leat a few test cases from some (semi-) official calculator (https://familienportal.de/familienportal/rechner-antraege/elterngeldrechner)

Reply:
This calculator requires Steuerklassen. We assume Steuerklasse 4 for everybody? Are there maybe test cases from an elterngeld implementation at ZEW @stichnoth or IFO @mjbloemer .

Reply:
It should work with Steuerklasse 4 in the absence of other deductions.

Reply:
As we do not have Lohnsteuer yet and just use a proxy for net income, we would pass a external test yet. However I included an example and we were close up to 10 Euro. I documented this the test file.

Reply:
Well, it is hard to tell with that calculator where the difference might be coming from, isn't it? Rather unclear where they might make different assumptions on Social Insurance contributions, ... ? So within 10 € is fine, I think.

Please double-check the other entries with the calculator, though, to the extent that this is possible

Reply:
I checked and we are quite close. I think we should do that again, once we have Lohnsteuer modeled.